src/backend/expungeservice/models/charge_types/contempt_of_court.py

```diff
@@ -0,0 +1,21 @@
+from dataclasses import dataclass
+
+from expungeservice.models.charge import Charge
+from expungeservice.models.expungement_result import TypeEligibility, EligibilityStatus
+
+
+
+@dataclass(eq=False)
+class ContemptOfCourt(Charge):
+    """ This is a civil offense """
+
+    type_name: str = "Contempt of Court"
+    expungement_rules: str = (
```

Comment:
This should be a longer-form string, i.e. including that fact that it may or may not be classifiable by charge level. The details here can depend on how we decide to handle violation/misdemeanor contempt charges 

Also it can note that statutes for a contempt charge include 33015, 33055, and 33065.